tutorials/standalone-tensorflow-raspberry-pi/index.md

```diff
@@ -0,0 +1,168 @@
+date_published: 2018-08-19
+---
+
+
+# Setting Up a Standalone Tensorflow Prediction Device with Raspberry Pi and Google Cloud
+
+This tutorial describes how to set up a Tensorflow server in a Raspberry Pi 3B and deploy on it a Tensorflow saved model. 
+
+The architecture of a machine learning model trained in the cloud and served locally is particularly useful when the IoT device has poor connectivity or when the serving availability is critical. 
+
+This tutorial assumes you have a Raspberry Pi with SD card and Raspbian installed on it, that you are familiar with Python and Tensorflow and that you have available a Tensorflow saved model. If you don’t have access to a saved model, you can alternatively use the saved model resulting from the Image Classification using Flowers dataset tutorial that is provided in this guide for download.
+
+## Challenges solved
+Tensorflow models can be served via Cloud ML Engine, Tensorflow Serving API, or with the tf.contrib.predictor library in Python. Since we aim for local prediction without dependency of a connection to the internet, we have discarded the first option. Due to the ARM architecture of the chip in the Raspberry Pi, installing the Tensorflow Serving API requires to first solve the challenges associated with compiling the solution in that architecture.
+The tf.contrib.predictor library in Python allows for serving the model in a straightforward fashion, and therefore we have chosen this method for the solution.
+
+This solution can be useful for applications in agricultural industries where there might be low connectivity to the internet or in other industries that cannot afford for the roundtrip of an API call in the internet. 
+
+## Objectives
+- Install Tensorflow 1.8 and its dependencies
+- Install software to capture images from a webcam
+- Download a Tensorflow saved model
+- Write a script to serve a Tensorflow model
+
+## Costs
+If you decide to train your own model, you will incur costs of running Cloud ML Engine, Dataflow, Cloud Storage. Otherwise, there are no billable costs involved in this tutorial.
+Use the Pricing Calculator to generate a cost estimate based on your projected usage.
+
+## Reference Architecture
+
+The following diagram shows the architecture of the solution
+
+![N|Solid](https://storage.googleapis.com/standalone-tensorflow-raspberry-pi/img/standalone_tensorflow_img1.png)
+
+1. Data as pictures, csv files, etc. is originally stored in Google Cloud Storage.
+1. The Tensorflow model training is executed in Cloud ML Engine
+1. The resulting saved model is stored in a bucket in Google Cloud Storage
+1. The saved model is transferred to the IoT Device
+1. Serving of the model (e.g. image recognition) is done locally in the IoT device
+
+## Before you begin
+1. Log into the Raspberry Pi
+
+1. Install required system packages.
+Several packages are required to successfully install gsutil from PyPI. You can install them with the following command:
+    ```
```

Comment:
Our publishing system doesn't deal well with codeblocks delimited by codefences of three backticks (```)---especially in lists. During the production pass, I can convert these codeblocks to use indentation, or you can do it yourself if you'd rather.

Reply:
I tried to change this, but it didn't look as I expected. I'll appreciate if you do this.